content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -504,6 +504,19 @@ termination grace period _begins_. The behavior above is described when the
 feature gate `EndpointSliceTerminatingCondition` is enabled.
 {{</note>}}
 
+{{<note>}}
+Starting from Kubernetes 1.29, if your Pod has one or more sidecar containers,
```

Comment:
End users may misunderstand the term "sidecar container" as it's been a while since we've called the regular container that supports the main container "the sidecar container".

Maybe it would be better to be written more clearly, like "an init container with Always restart policy".